platform/ui-next/src/components/StudyItem/StudyItem.tsx

```diff
@@ -4,6 +4,8 @@ import classnames from 'classnames';
 import { ThumbnailList } from '../ThumbnailList';
 
 import { Accordion, AccordionContent, AccordionItem, AccordionTrigger } from '../Accordion';
+import { Tooltip, TooltipContent } from '../Tooltip';
+import { TooltipTrigger } from '@radix-ui/react-tooltip';
```

Comment:
I think this is wrong, you should grab the TooltipTrigger from our UI not from radix, no?